ui/decredmaterial/scrollbar.go

```diff
@@ -0,0 +1,163 @@
+package decredmaterial
+
+import (
+	"image"
+	"image/color"
+
+	"gioui.org/f32"
+	"gioui.org/gesture"
+	"gioui.org/io/pointer"
+	"gioui.org/layout"
+	"gioui.org/op"
+	"gioui.org/op/clip"
+	"gioui.org/op/paint"
+	"gioui.org/unit"
+	"gioui.org/widget"
+)
+
+type Scroller struct {
+	clickable widget.Clickable
+	drag      gesture.Drag
+	scrolled  bool
+	length    int
+	progress  float32
```

Comment:
kindly add comment to this, "progress" isn't quite clear. We already have a progress widget, I can't relate anything that has to do with a scrollbar with the word "progress"